Python-maxsubarray/README.md

```diff
@@ -0,0 +1,22 @@
+<h1>Palindrome Lab</h1>
+
+<h2>Assessment Language</h2>
+Python
+
+<h2>Autograder Language</h2>
+Python
+
+<h2>Autograding Environment Packages</h2>
+Python3 and pip are needed for the autograding environment.
```

Comment:
`pip` can get installed alongside `python3` but I don't think it is actually needed for your autograding environment, so consider removing it from the list here.